cc3d/player5/Plugins/ViewManagerPlugins/DemoBrowser.py

```diff
@@ -12,9 +12,228 @@
 from cc3d.player5.styles.SyntaxHighligher import *
 
 MISSING_FILE_MESSAGE = "This demo doesn't have that kind of file available to preview."
-DESCRIPTION_FNAME = 'README.txt'
-INDEX_FILE_NAME = '.demoindex'
-DELIMITER = ','
+DESCRIPTION_FNAME = "README.txt"
+INDEX_FILE_NAME = ".demoindex"
+DELIMITER = ","
+
+
+class DemoBrowser(QDialog, ui_demo_browser.Ui_demoDialog):
```

Comment:
For this file, I might have missed some changes since the diff is large. It looks like the main changes are formatting and a wait popup. It works for me.

Reply:
Yes , I did black formatting so this is why there were a lot of changes but you caught all that matters. Thanks for looking into it!